.github/workflows/codeql-codescan.yml

```
@@ -31,11 +32,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'typescript']
+        language: [ 'typescript' ]
 
     steps:
     - name: Checkout repository
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)